tests/integration/component.go

```diff
@@ -236,6 +236,24 @@ func componentTests(args ...string) {
 			helper.CmdShouldPass("odo", append(args, "delete", "-f", "--all", "--context", context)...)
 		})
 
+		It("should describe not pushed component when it is created with json output", func() {
+			cmpDescribeJSON, err := helper.Unindented(helper.CmdShouldPass("odo", append(args, "create", "nodejs", "cmp-git", "--project", project, "--git", "https://github.com/openshift/nodejs-ex", "--context", context, "--app", "testing", "-o", "json")...))
```

Comment:
Can you use nodejs local source (less time consuming) instead of git source.

Reply:
fixed